core/trino-main/src/main/java/io/trino/server/protocol/spooling/SpoolingConfig.java

```diff
@@ -149,4 +118,13 @@ public boolean isSharedEncryptionKeySet()
     {
         return sharedEncryptionKey.isPresent();
     }
+
+    public enum SegmentRetrievalMode
```

Comment:
Do you have doc for that already?

Reply:
Not yet, I'm working with Manfred on the docs